src/main/flight/pid.h

```diff
@@ -177,6 +177,7 @@ PG_DECLARE_ARRAY(pidProfile_t, PID_PROFILE_COUNT, pidProfiles);
 
 typedef struct pidConfig_s {
     uint8_t pid_process_denom;              // Processing denominator for PID controller vs gyro sampling rate
+    uint8_t pid_adjust_aux_channel;              // RC channel-No (0-3:off / 4-7:AUX1-AUX4) used to adjust PID according to the flight mode (helicopter like vehicles)
```

Comment:
Please see parameter group documentation.
New values must be added to end of group to maintain compatibility.

Reply:
What about the conditional values (USE_RUNAWAY_TAKEOFF)?
I have been unsure whether to add the new value before or after the optional values.
So i have to put it to the end?

Reply:
Added new value to the end!